main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -513,8 +513,8 @@ class Flix {
       afterDesugar = Desugar.run(afterWeeder, cachedDesugarAst, changeSet)
       afterNamer <- Namer.run(afterDesugar)
       afterResolver <- Resolver.run(afterNamer, cachedResolverAst, changeSet)
-      afterKinder <- Kinder.run(afterResolver, cachedKinderAst, changeSet)
-      afterDeriver <- Deriver.run(afterKinder)
+      (afterKinder, kinderErrors) = Kinder.run(afterResolver, cachedKinderAst, changeSet)
+      afterDeriver <- Deriver.run(afterKinder).withSoftFailures(kinderErrors)
```

Comment:
This call `withSoftFailure` is temporary-- it is only needed as long as the pipeline is monadic.